pkg/controllers/restore/controller.go

```diff
@@ -472,30 +503,51 @@ func (h *handler) generateDependencyGraph(ownerToDependentsList map[string][]res
 }
 
 func (h *handler) createFromDependencyGraph(ownerToDependentsList map[string][]restoreObj, created map[string]bool,
-	numOwnerReferences map[string]int, objFromBackupCR ObjectsFromBackupCR, toRestore []restoreObj) error {
+	numOwnerReferences map[string]int, objFromBackupCR ObjectsFromBackupCR, toRestore []restoreObj, restoreWebhooks bool) ([]restoreObj, error) {
 	numTotalDependents := 0
 	for _, dependents := range ownerToDependentsList {
 		numTotalDependents += len(dependents)
 	}
+
+	mutatingWebhookGVR := schema.GroupVersionResource{
+		Group:    arv1.SchemeGroupVersion.Group,
+		Version:  arv1.SchemeGroupVersion.Version,
+		Resource: mutatingWebhookConfigurationResource,
+	}
+	validatingWebhookGVR := schema.GroupVersionResource{
+		Group:    arv1.SchemeGroupVersion.Group,
+		Version:  arv1.SchemeGroupVersion.Version,
+		Resource: validatingWebhookConfigurationResource,
+	}
+
 	countRestored := 0
 	var errList []error
+	var webhookList []restoreObj
 	for len(toRestore) > 0 {
 		curr := toRestore[0]
 		if len(toRestore) == 1 {
 			toRestore = []restoreObj{}
 		} else {
 			toRestore = toRestore[1:]
 		}
-		if created[curr.ResourceConfigPath] {
-			logrus.Infof("Resource %v is already created/updated", curr.ResourceConfigPath)
-			continue
-		}
```

Comment:
Since we are using every field anyway, I moved this so that `currResourceInfo` is created first. This should improve readability for future committers.